wurst/file/FileIO.wurst

```diff
@@ -13,14 +15,12 @@ import ErrorHandling
 	>	new File("MyFileName.pld")..write("MyContent")..close()
 */
 
-/** Abilities which tooltips will be used to load data */
-@configurable constant IO_ABILITIES = [AbilityIds.aerialShackles, AbilityIds.rocketAttack, AbilityIds.callToArms,
-						AbilityIds.calltoArms, AbilityIds.cloudofFog, AbilityIds.controlMagic,
-						AbilityIds.defend, AbilityIds.dispelMagic, AbilityIds.feedbackArcaneTower,
-						AbilityIds.feedback]
+@configurable public constant CHUNKS_PER_FILE = 64
 
 public var CAN_READ_FILES = true
 
+constant DUMMY_ABIL_ID = compiletime(ABIL_ID_GEN.next())
```

Comment:
This should be a hardcoded id. Otherwise the id may change when you make a new version of the map (adding some abilities), and you won't be able to read the data you wrote in previous versions.

Reply:
good point